definitions/scenarios/upgrade_to_satellite_6_15.rb

```diff
@@ -38,7 +67,6 @@ class PreMigrations < Abstract
     end
 
     def compose
-      add_steps(find_procedures(:pre_migrations))
```

Comment:
@evgeni Your comment about maintenance mode made me realize that due to the magic of foreman-maintain's tags, I totally removed maintenance mode by removing this line.

Reply:
what? :grin: